wifiphisher/common/macmatcher.py

```diff
@@ -57,6 +57,30 @@ def _get_vendor_information(self):
                     logo = separated_line[2]
                     self._mac_to_vendor[mac_identifier] = (vendor, logo)
 
+    def get_random_mac_addrs(self, num_of_mac_addrs):
```

Comment:
Also it would be better to move this to the `interfaces` module as a function there.